controllers/authorization.controller.js

```diff
@@ -0,0 +1,56 @@
+import config from "config";
+
+import {
+  generateAccessToken,
+  generateRefreshToken,
+} from "../utils/tokenGenerator.js";
+import { compareHashes } from "../utils/compareHashes.js";
+
+import { User } from "../model/user.js";
+
+const accessCookieOptions = config.get("cookie.accessCookieOptions");
+const refreshCookieOptions = config.get("cookie.refreshCookieOptions");
+const weatherAccessCookie = config.get("cookie.weatherAccessCookie");
+const weatherRefreshCookie = config.get("cookie.weatherRefreshCookie");
+
+export class AuthorizationController {
+  static async authCheck(req, res) {
+    res.status(200).send();
+  }
+
+  static async logout(req, res) {
+    res.cookie(weatherAccessCookie, "", { ...accessCookieOptions, maxAge: 0 });
+    res.cookie(weatherRefreshCookie, "", {
+      ...refreshCookieOptions,
+      maxAge: 0,
+    });
+    res.status(200).send();
+  }
+
+  static async login(req, res) {
+    const { username, password } = req.body;
+
+    User.findOne({ username }).then((user) => {
+      if (!user) {
+        res.status(404).json({ message: "User doesn't exist" });
+      } else {
```

Comment:
Avoid if {} else {} constructions, too much indents makes code more complicated
if (!user) {
        res.status(404).json({ message: "User doesn't exist" });
	return;
}
// other code

Reply:
If user doesnt exist - return 403 :)) Do you know why returning 404 is bad solution?